### PR TITLE
6 allow resource init and cleanup

### PR DESCRIPTION
**Pull request by TNO-SlaFleur**

### CODE REVIEW

File: src/nwnsdk/nwn_client.py [hunk 1]
```diff
@@ -74,3 +82,6 @@ def db_client(self) -> PostgresClient:
     @property
     def broker_client(self) -> RabbitmqClient:
         return self.rabbitmq_client
```
Comment (MarkTNO) [inline src/nwnsdk/nwn_client.py:84]: Could these two properties be deleted? It would be better if only nwn_client functions are called.

Comment (TNO-SlaFleur) [reply src/nwnsdk/nwn_client.py:84]: That is okay, but then I will refactor NWNClient to use RabbitMQClient and PostgresClient as mixins. That way, we do not have to write so many useless functions that only call the same function on the proper client. Will save us a lot of hassle ^^

**Comment by TNO-SlaFleur** [comment]
Mixins can be complicated but considering we 2 are working on them I think we can manage. If you have any questions on how to approach them, please let me know. Here is a nice resource if you just want to read up on them a bit: https://realpython.com/inheritance-composition-python/

**Comment by MarkTNO** [comment]
Nice solution!

**Pull request merged**